CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.1.1
+
+### Bug fixes
+- Fix hardcoded CRD version in crd scaffold ([#690](https://github.com/operator-framework/operator-sdk/pull/690))
+- Remove unnecessary TODO in `cmd/manager/main.go` ([#679](https://github.com/operator-framework/operator-sdk/pull/679))
```

Comment:
We can remove this from the CHANGELOG. The key fix here is the hardcoded CRD version.
The TODO isn't a significant enough change to mention it in the CHANGELOG. Plus it's not a bug fix.